The libsndfile.so library does not have the architecture in the name and thus would not be
found, consequently _soundfile_data would not be build. Removing the 'architecture0' bit
fixes that
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/python-soundfile-0.12.1.orig/setup.py b/python-soundfile-0.12.1/setup.py
index ef99bcc..8a44a82 100644
--- a/python-soundfile-0.12.1.orig/setup.py
+++ b/python-soundfile-0.12.1/setup.py
@@ -21,7 +21,7 @@ if platform == 'darwin':
 elif platform == 'win32':
     libname = 'libsndfile_' + architecture0 + '.dll'
 elif platform == 'linux':
-    libname = 'libsndfile_' + architecture0 + '.so'
+    libname = 'libsndfile' + '.so'
 else:
     libname = None
 
